[2025-05-05T07:23:08.864+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-05T07:23:08.892+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load_task scheduled__2025-05-04T00:00:00+00:00 [queued]>
[2025-05-05T07:23:08.905+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load_task scheduled__2025-05-04T00:00:00+00:00 [queued]>
[2025-05-05T07:23:08.906+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-05-05T07:23:08.925+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_task> on 2025-05-04 00:00:00+00:00
[2025-05-05T07:23:08.934+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'load_task', 'scheduled__2025-05-04T00:00:00+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp7pj_gfjh']
[2025-05-05T07:23:08.937+0000] {standard_task_runner.py:105} INFO - Job 175: Subtask load_task
[2025-05-05T07:23:08.937+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=175) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-05T07:23:08.938+0000] {standard_task_runner.py:72} INFO - Started process 176 to run task
[2025-05-05T07:23:08.991+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.load_task scheduled__2025-05-04T00:00:00+00:00 [running]> on host e5bea2914fd7
[2025-05-05T07:23:09.088+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-04T00:00:00+00:00'
[2025-05-05T07:23:09.089+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-05T07:23:09.114+0000] {etl_pipeline.py:45} INFO - Données à charger : [['2025-05-05', 6.4, 14.1, 0.0, 80], ['2025-05-06', 6.6, 15.6, 0.1, 61], ['2025-05-07', 8.5, 16.8, 0.0, 3], ['2025-05-08', 8.8, 16.4, 1.2, 80], ['2025-05-09', 10.5, 15.8, 0.0, 3], ['2025-05-10', 9.0, 20.6, 0.0, 3], ['2025-05-11', 9.7, 21.9, 0.0, 2]]
[2025-05-05T07:23:09.125+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-05T07:23:09.137+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-05T07:23:09.138+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load_task, run_id=scheduled__2025-05-04T00:00:00+00:00, execution_date=20250504T000000, start_date=20250505T072308, end_date=20250505T072309
[2025-05-05T07:23:09.194+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-05T07:23:09.213+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-05T07:23:09.215+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
